### ISSUE TITLE
Domains: Create a way to obtain A Records while connected via name servers

### ISSUE THREAD

**Issue by cuemarie** [Automattic/wp-calypso#72105]
### What

Enable a way to open the connection setup instructions and obtain A & CNAME records while a domain is actively connected via Name servers. 

### Why

If a domain is connected via Name Servers (our recommended method), then the only way to switch that connection to the alternative method is to first break the connection, so that Calypso provides steps to obtain the IP addresses and CNAME record needed to connect the domain again. (Beginning of process shown below): 

![screenshot](https://en-support.files.wordpress.com/2022/04/connect-domain-review-setup-instructions.png)
*Don't see an image above? [Click here](https://en-support.files.wordpress.com/2022/04/connect-domain-review-setup-instructions.png)*

If a user needs to move from a name servers-based connection to an A record connection, their only paths forward are to break their current connection, disrupting their site, or contact support so we can help them more manually. As my chatter in 26097311-hc noted:

> Oh. So I have to break that configuration first, so access to the site on the domain stops working and then go through the process again?

> Ok. That is unfortunate. I'll work on getting this handled, then. I would recommend as a feature to be added, the ability for users to get that information, without having to break their site and disrupt access to the people they service.

### How

Provide a link to the "Advanced setup" of a domain somewhere in Calypso, even while the domain is actively connected by name servers.

**Comment by delputnam** [comment]
This is a good suggestion.  I'll add this to a future sprint for Nomado.  We'll update here once we've made an update.

**Comment by delputnam** [comment]
FYI, the workaround for this is to let the user know what the A records should be for their domain, then they can add those records at their new name server...and then switch the name servers at their registrar.
